lib/backend_common/backend_common/auth.py

```diff
@@ -116,6 +116,35 @@ def has_permissions(self, permissions):
         return taskcluster.utils.scopeMatch(self.get_permissions(), permissions)
 
 
+class Auth0User(BaseUser):
+
+    type = 'auth0'
+
+    def __init__(self, token, userinfo):
+        assert 'email' in userinfo
```

Comment:
I may sound a bit paranoid, but can you avoid using `assert`. It always reminds me about [this issue](https://github.com/IdentityPython/pysaml2/issues/451) :)